### ISSUE TITLE
Suggestion: Start with two groups open

### ISSUE THREAD

**Issue by DatGrey** [projectdelphai/panorama-tab-groups#65]
Users that have the "Restore previous session" option disabled have to manually create (at least) a second group every time they launch Firefox; having it launch with two groups already open would make the users able to Alt-W right away and open tabs in different groups if needed.

**Comment by vaxul** [comment]
@DatGrey 
Hi, I try to understand your suggestion. Could you provide us some steps to reproduce your usecase, please?

Please try to explain step by step what do you try, what happen in the current state and what do you expect to happen if it was like in your suggestion.

F.e. if I have the option `Restore previous session` disabled in Firefox

1. Open a tab the first group
2. Create a second group
3. Open a tb in the second group
4. If I close Firefox, I get a prompt which ask to confirm, that I want to close 3 open tabs.

Current behaviour:
If I confirm the prompt Firefox closes the window with all tabs.
If I restart Firefox all tabs are gone (including groups). Which is the expected behaviour if you disable the option `Restore previous session`.

Best regards

**Comment by DatGrey** [comment]
@vaxul take a look at issue #52 too, the suggestion is kind of an alternative to the functionality described there; the behaviour I'd like to see is:
1. Have Firefox open with various tabs divided by groups
2. Close Firefox; all tabs and groups are lost because the "Restore previous session" option is disabled, and that's expected
3. Re-open Firefox; it starts with two groups already active, so the user can start browsing and opening tabs in different groups just by pressing Alt+W instead of having to first open the addon page with Ctrl+Shift+F, double clicking on an empty space, dragging the newly created group away from the first one (it gets created on top of it) and THEN start using the two groups.
I know it's just a minor usability upgrade, but I think it would be neat and streamline the usage of the addon a lot.

**Comment by vaxul** [comment]
Sorry, I still not sure if I understand it. I read your text twice. 

Your ask to add an option to PTG to start with more then one **empty** tab groups by default?

**Comment by DatGrey** [comment]
Yes, that's what I'm asking for. Sorry for not being very clear, English is not my native language.

**Comment by vaxul** [comment]
Ok, no problem. 

As far as I can overlook the code, this should be possible. I look into it.

**Comment by DatGrey** [comment]
Thanks a lot!

**Comment by vaxul** [comment]
I tried at out:
https://github.com/vaxul/panorama-tab-groups/blob/feature/65_allow_multiple_empty_groups_at_start/src/js/background.js#L228-L242

It:
* Adds an option the addon options page which allow to set the number of groups up to 5 at a clean start, with the default 1.
* Creates multiple groups at startup if set in the options.

Very basically it works, **but**:
* the multiple groups are currently created only after the first switch to the PTG view, so no ALT+W at startup in the moment
* the 2nd group isn't placed properly

Have someone any suggestions on how to solve the both issues in an elegant way?

**Comment by vaxul** [comment]
@DatGrey 
Could you please test https://github.com/vaxul/panorama-tab-groups/tree/feature/65_allow_multiple_empty_groups_at_start 

Would this current state (see previous comment) helpful?

**Comment by DatGrey** [comment]
@vaxul sorry for the late reply, I've tried it out and it seems to be working as intended! Having the second group already active at startup would make the functionality perfect but it's a minor nitpick at this point, this change would be a big improvement already and it's perfectly usable as is. Thanks a lot!

**Comment by vaxul** [comment]
Added pull request for the feature.
